tests/test_schema.py

```diff
@@ -158,9 +164,9 @@ def test_include_data_with_single(self, post):
         assert 'first_name' in author['attributes']
 
     def test_include_data_with_all_relations(self, post):
-        data = PostSchema(include_data=('author', 'post_comments')).dump(post).data
+        data = PostSchema(include_data=('author', 'post_comments', 'post_comments.author')).dump(post).data
         assert 'included' in data
-        assert len(data['included']) == 3
+        assert len(data['included']) == 5
```

Comment:
I'm note quite sure if that's the assertion you mean...If that's not correct, can you be a bit more specific?

Also note the call of str(), shouldn't be the id field always be string (see #48)?